src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -305,6 +305,8 @@ void GcsPlacementGroupManager::RemovePlacementGroup(
     const PlacementGroupID &placement_group_id,
     StatusCallback on_placement_group_removed) {
   RAY_CHECK(on_placement_group_removed);
+  RAY_LOG(INFO) << "Removing placement group " << placement_group_id;
```

Comment:
The printing frequency of these logs is very low. In order to facilitate the problem location, it is set to `INFO`.

Reply:
Hmm I still feel like this log is unnecessary. Isn't it better adding a log when the placement group is actually deleted instead?

Reply:
Fixed